uint16 port
string return_address
uint16 return_port
string source_topic
string destination_topic
uint32 queue_size
float32 period
